.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "endOfLine": "auto"
```

Comment:
Windows prettier will change linefeed in a way that blows up without this.

Reply:
Yeah I was seeing that here: https://github.com/prettier/prettier/pull/5327, really good catch!